product/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from .views import ProductListView, FilterView
+
+urlpatterns = [
+    path('/allitem-list', ProductListView.as_view()),
+    path('/allitem-filter', FilterView.as_view())
```

Comment:
allitem 엔드포인트 URL 이 좀 어색한거같아서 아래와같이 수정 부탁 드립니다. 
이미 product url로 라우팅 되고 있기 때문에 list 와 모든 item 이라는 것이 꼭 들어가야 하나 싶습니다.
```suggestion
    path('', ProductListView.as_view()),
    path('/filter', FilterView.as_view())
```

Reply:
수정하였습니다.